src/main/java/com/immunization/reference/model/ConnectionInfo.java

```diff
@@ -0,0 +1,41 @@
+package com.immunization.reference.model;
+
+public class ConnectionInfo {
+
+  private String userId;
+  private String password;
+  private String facility;
+  private String path;
```

Comment:
This is supposed to be a valid URL for sending the HL7 message, right?

Reply:
Yup, should we rename it to like iisUrl or something?

Reply:
Sure.

Reply:
done